aws-go-fargate/main.go

```diff
@@ -124,8 +124,8 @@ func main() {
 
 		// Get credentials for the new ECR repository
 		repoCreds := repo.RegistryId.ApplyT(func(rid string) ([]string, error) {
-			creds, err := ecr.GetCredentials(ctx, &ecr.GetCredentialsArgs{
-				RegistryId: rid,
+			creds, err := ecr.GetAuthorizationToken(ctx, &ecr.GetAuthorizationTokenArgs{
```

Comment:
With `GetAuthorizationToken` you don't have to decode the output anymore, username and password are available as fields.